renderdoc/api/replay/data_types.h

```diff
@@ -1423,6 +1423,18 @@ worked around by re-sorting bindings.
 )");
   bool shadersMutable = false;
 
+  DOCUMENT("(``True`` if the API supports vertex shader debugging.");
+  bool vertexShaderDebugging = false;
+
+  DOCUMENT("(``True`` if the API supports pixel shader debugging.");
+  bool pixelShaderDebugging = false;
+
+  DOCUMENT("(``True`` if the API supports compute shader debugging.");
+  bool computeShaderDebugging = false;
```

Comment:
I think on balance I'd prefer just to have a single `shaderDebugging` feature bit here. I don't think we should allow backends to only support certain types, at least for these three that we currently support.